frontend/src/views/Home.vue

```diff
@@ -66,6 +67,14 @@ export default {
   width: 100%;
 }
 
+.home > h1 {
+  margin-bottom:2vh;
```

Comment:
?

Reply:
Default is too high

Reply:
2vh seems high where is this coming from I don't think I saw it.

Reply:
Sorry, this should definitely be in .splash, i'll move it.
It's just the spacing between the logo and line of text underneath, I'll use a pixel amount mebe?
Default h1 spacing is humungous, that's all.

Reply:
We should override h1, h2, h3, h4 to all inherit from font at some point.

Reply:
em is probably the go.